R/reducolor.R

```diff
@@ -1,4 +1,4 @@
-#' Reduce image color to either 2 or 8 colors for cartoonized effect
+library(devtools)#' Reduce image color to either 2 or 8 colors for cartoonized effect
```

Comment:
Not sure why devtools is attached here?

```suggestion
#` Reduce image color to either 2 or 8 colors for cartoonized effect
```

Reply:
Good Catch!